### DISCUSSION TITLE
The weights of splade

### DISCUSSION THREAD

**Discussion by Wenze7** [castorini/pyserini#1363] [Q&A] [answered]
I download the following file from "topics-and-qrels/topics.msmarco-passage.dev-subset.distill-splade-max.tsv.gz"
<img width="1369" alt="image" src="https://user-images.githubusercontent.com/55132749/204308940-4eee524b-8827-4037-9cd4-f7a5a905c19c.png">
I want to know where are the weights produced by splade? And i don't know why many tokens are repeated?

**Comment by lintool** [comment, marked as answer]
The weight is equal to the number of times the token is repeated, e.g., if "it" appears five times, it has a weight of five.

**Reply by Wenze7** [reply]
Thanks very much!
